tests/commonlib/utils.py

```diff
@@ -30,14 +30,15 @@ def get_ES_evaluation(
 
     while time.time() - start_time < timeout:
         try:
+            time.sleep(2)
```

Comment:
why are you adding sleep? can you please document or remove it?

Reply:
This timeout used for reducing requests frequency to ElasticSearch. I added comment.